inc/Engine/Optimization/RUCSS/Admin/Subscriber.php

```diff
@@ -272,4 +298,41 @@ public function clear_usedcss_result() {
 	public function add_clean_used_css_menu_item( $wp_admin_bar ) {
 		$this->settings->add_clean_used_css_menu_item( $wp_admin_bar );
 	}
+
+	/**
+	 * Displays the RUCSS progressbar
+	 *
+	 * @since 3.9
+	 *
+	 * @param string $field_id ID of the settings field.
+	 *
+	 * @return void
+	 */
+	public function display_progress_bar( $field_id ) {
+		if ( 'remove_unused_css' !== $field_id ) {
+			return;
+		}
+
+		$this->settings->display_progress_bar();
+	}
+
+	/**
+	 * Array with UI translations.
+	 *
+	 * @return array
+	 */
+	private function ui_translations() : array {
+		return [
+			'scanning'          => __( 'Scanning', 'rocket' ),
+			'from'              => __( 'from', 'rocket' ),
+			'in'                => __( 'in', 'rocket' ),
+			'seconds'           => __( 'seconds', 'rocket' ),
+			'warming_resources' => __( 'Warming resources', 'rocket' ),
+			'rucss_working'     => __( 'RUCSS working!', 'rocket' ),
+			'rucss_btn'         => __( 'Generate Used CSS now', 'rocket' ),
+			'warmed_list'       => __( 'Not warmed resources list:', 'rocket' ),
+			'warming_resources' => __( 'Warming', 'rocket' ),
+			'warming_resources' => __( 'Warming', 'rocket' ),
```

Comment:
Individual translation is not good, because translations and meaning can change depending on the language and context.

Ideally, we should provide full sentences to be translated, with placeholders for the part that needs to be filled in programatically.

Reply:
@Tabrisrp great idea! I modified the translations and I added placeholders.